src/StudioCore/Assets/Paramdex/ER/Defs/AtkParam.xml

```diff
@@ -1397,7 +1397,12 @@
     </Field>
     <Field Def="s32 finalDamageRateId">
     </Field>
-    <Field Def="dummy8 pad7[12]">
-    </Field>
```

Comment:
This dummy I'm fairly sure came after finaldamagerate, so these could just be renamings.
Also it may be worth checking if finaldamagerate was this as well, since it showed up in like 1.06 or 1.07 and wasn't in original paramdefs